tx_prelude/src/lib.rs

```diff
@@ -245,6 +248,15 @@ impl StorageWrite for Ctx {
         unsafe { namada_tx_delete(key.as_ptr() as _, key.len() as _) };
         Ok(())
     }
+
+    // TODO: is this the correct thing to do here?
```

Comment:
I'm not too sure, but we might need to be able to choose this from wasm txs too, then we might need to pass `WriteActions` back to host

Reply:
done in b7b1921cf0a8102a73bc75104dfbb29735a38a8c